src/Aspire.Hosting.GitHub.Models/api/Aspire.Hosting.GitHub.Models.cs

```diff
@@ -43,6 +43,9 @@ public static partial class Cohere
 
         public static partial class Core42
         {
+            public static readonly GitHubModel Jais30bChat;
+            [System.Obsolete("Use Jais30bChat instead.")]
+            [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
@sebastienros - same question here about Obsolete properties in a library that hasn't shipped stable (yet - it will in 13).